controllers/runner_controller.go

```diff
@@ -78,6 +80,11 @@ type RunnerReconciler struct {
 // +kubebuilder:rbac:groups=actions.summerwind.dev,resources=runners/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,resources=pods/finalizers,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=core,resources=pods/exec;rolebindings,verbs=get;create
+// +kubebuilder:rbac:groups=core,resources=pods/log;rolebindings,verbs=get;list;watch
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles;rolebindings,verbs=get;list;watch;create;delete
+// +kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;watch;create;delete
+// +kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;create;delete
```

Comment:
In order to create roles for the runner with these permissions, our ClusterRole must have at least the same roles and possibly more. In other words, I cannot assign `batch jobs` role for the runner if the ARC managers ClusterRole does not have the permission as well